src/components/Reportees/index.jsx

```diff
@@ -10,10 +10,16 @@ import CONFIG from '../../config';
 
 const sortByPersonName = (a, b) => (a.cn <= b.cn ? -1 : 1);
 
+const styles = {
+  root: {
+    maxWidth: 500,
+  },
+};
+
 const Reportees = ({
-  ldapEmail, partialOrg, metrics,
+  classes, ldapEmail, partialOrg, metrics,
 }) => (
-  <Grid>
+  <div className={classes.root}>
```

Comment:
Instead of adding the maxWidth to the Reportees view, add it to the main content of the app as a whole so that it gets applied for every view. Note that you will still want the navigation bar to span the full length.